AppCUI/include/AppCUI.hpp

```diff
@@ -892,7 +892,7 @@ namespace Utils
     };
     class EXPORT NumericFormatter
     {
-        char temp[72]; // a minimum of 65 chars must be allocated to support 64 bits for binary translation
+        char temp[72]{}; // a minimum of 65 chars must be allocated to support 64 bits for binary translation
```

Comment:
acel {} forteaza un memset la initializare cu 0 --> care cumva face un delay (mai ales daca initializezi intr-o bucla;
In practica nu prea conteaza pentru ca se suprascrie mereu si se da un string_view strict pe ce s-a suprascris --> este sa zicem by design facut asa.

Reply:
Ok, doar genera warnings.